socketio_middleware/src/rooms.rs

```diff
@@ -96,13 +100,24 @@ pub fn get_sockets_number_for_room(room_id: &str) -> usize {
 pub fn print_sockets_for_room(room_id: &str) {
     match ROOMS.get(room_id) {
         Some(sockets) => {
+            info!("ROOMS: room {} containt sockets number = {}.", room_id, sockets.len());
+            /*
             for socket in &*sockets {
                 debug!("ROOMS: room {} containted socketid {}, sockets number = {}.", room_id, socket.sid(), sockets.len());
             }
+            */
         }
 
         None => {
             debug!("ROOMS: no socket in room {}.", room_id);
         }
     }
 }
+
+///
+/// get rooms number
```

Comment:
A little more descriptive comment would be great, something like "Returns the number of rooms this middleware currently has reference too."

Reply:
NP